server/sonar-web/src/main/js/apps/issues/sidebar/TypeFacet.js

```diff
@@ -35,17 +36,18 @@ type Props = {|
   stats?: { [string]: number },
   types: Array<string>
 |};
+*/
 
 export default class TypeFacet extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
   static defaultProps = {
     open: true
   };
 
   property = 'types';
 
-  handleItemClick = (itemValue: string) => {
+  handleItemClick = (itemValue /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)